reefer_monitoring/v1/reefer_v1.0.0-Beta-1.yaml

```diff
@@ -0,0 +1,524 @@
+    controllerSerialNumber:
+      type: string
+      x-stoplight:
+        id: 85198024f8ff2
+      description: The serial number of the controller device
+    controllerSoftwareVersion:
+      type: string
+      x-stoplight:
+        id: 6c473aa909c95
+      description: The software version running on the controller device
+    currentPhaseA:
+      type: number
+      x-stoplight:
+        id: 8a465afd81161
+      format: float
+      description: The phase A current or average current in Amps
+    currentPhaseB:
+      type: number
+      x-stoplight:
+        id: 8a765afd81a78
+      format: float
+      description: The phase B current in Amps
+    currentPhaseC:
+      type: number
+      x-stoplight:
+        id: 8a665afd81d8f
+      format: float
+      description: The phase C current in Amps
+    defrosting:
+      type: boolean
+      x-stoplight:
+        id: 458de0f8230ef
+      description: Indicates if the device is defrosting
+    dischargePressure:
+      type: number
+      x-stoplight:
+        id: 49290aca593bd
+      format: float
+      description: The discharge pressure in BarRelative units
+    dischargeTemperature:
+      allOf:
+        - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/3.0.0#/components/schemas/measuredTemperature'
+      x-stoplight:
+        id: c404ba92de724
+      description: The discharge temperature
+    efficiencySetting:
+      type: number
+      x-stoplight:
+        id: a1beb5769dd83
+      description: The efficiency Setting (controller specific) COA UDM List 5.5.8
+    evaporatorTemperature:
+      type: number
+      x-stoplight:
+        id: 6272d1f6f40b4
+      format: float
+      description: The evaporator temperature in CEL
+    equipmentOptions:
+      type: array
+      x-stoplight:
+        id: bf9a1437488d6
+      description: List of equipment options available on the container
+      items:
+        type: string
+    geoLocation:
+      type: object
+      x-stoplight:
+        id: cc01ff8a61d52
+      properties:
+        latitude:
+          type: number
+          example: 48.85855
+        longitude:
+          type: number
+          example: 2.294492036
+    lastDefrost:
+      type: string
+      x-stoplight:
+        id: 2a7068aedfe2d
+      format: date-time
+      description: The end time of the past defrost period
+    lineFrequency:
+      type: number
+      x-stoplight:
+        id: 6e4b52aa20d93
+      format: float
+      description: The mains power frequency in Hz
+      example: 50
+    lineVoltage:
+      type: number
+      x-stoplight:
+        id: 61ca68123fe58
+      format: float
+      description: The mains power voltage in volts
+      example: 400
+    returnProbe1Temperature:
+      allOf:
+        - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/3.0.0#/components/schemas/measuredTemperature'
+      x-stoplight:
+        id: 6934ad00380e2
+      description: The return temperature of probe 1
```

Comment:
This is not allowed when using code generators - it is not allowed to "extend" the description like this - you have to duplicate the attribute :-(